nbgrader/docs/source/user_guide/release/ps1/problem2.ipynb

```diff
@@ -1,31 +1,5 @@
 {
  "cells": [
-  {
```

Comment:
Same comment here about missing cells

Reply:
Replaced with their original version from the main repository to address this